lib/new-rule.js

```diff
@@ -189,19 +189,16 @@ async function addRule(options, elmJson, ruleName, ruleType) {
   }
 
   // Inject configuration into every preview configuration
-  glob
-    .sync(OsHelpers.makePathOsAgnostic(`${dir}/preview*/**/elm.json`), {
+  for (const configurationElmJson of glob.sync(
```

Comment:
Let's extract `glob.sync` to a variable, this is a bit hard to read.

Reply:
`glob.sync`, or the whole expression?